testapi.pm

```diff
@@ -907,12 +907,12 @@ Wrapper around script_output, that runs a callback on the output. Use it as
 
 =cut
 
-sub validate_script_output($&;$) {
+sub validate_script_output {
```

Comment:
good but unrelated. Please put into at least another commit if not PR where you try to remove the signature for all sub's where possible